main.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# Line 1 adds shebang to make it work like a shell script 
+import argparse
+
+def main():
+    print("This is my first IGVC assingment regarding argparse.")
+    parser = argparse.ArgumentParser(description="This is an argparse")
+    parser.add_argument('-i', '--input')
```

Comment:
Look into the [`add_argument()`](https://docs.python.org/3/library/argparse.html#the-add-argument-method) documentation to set defaults and the appropriate types. Look into using [`pathlib`](https://docs.python.org/3/library/pathlib.html).